k8s/config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: vineyardcloudnative/vineyard-operator
+  newName: localhost:5001/vineyard-operator
```

Comment:
Don't use internal/testing configuration for default configuration.

The `k8s/config/manager/kustomization.yaml` should be enduser-oritend. You could tweak those options while `kustomize` in Makefile.

Reply:
The error is because the execution of an e2e test was interrupted in the middle and the operations to restore these images haven't been performed.